---
title: 'S-PrediXcan'
date: ''
categories: []
tags: []
weight: 2
---

### Scaling up PrediXcan to integrate large-scale data

Genetic studies have discovered many thousands of locations on chromosomes that are associated with health or disease, but the majority of these fall outside of protein-encoding regions, and so it is difficult to understand just how they exert their influences. PrediXcan leverages large scale DNA, observable trait, and gene expression datasets, as well as machine learning, to infer from these discoveries which genes are actually influenced and whether an up or downregulation of the gene is beneficial. Despite the appeal and success of this approach, which directly identifies candidate drug targets, accumulating experience revealed that data from over a million individuals would be required in many cases. We therefore developed an approach that bypasses the need to use individual level data and takes advantage of summary results, to vastly expand the applicability and power of the original PrediXcan method. The new approach, called S-PrediXcan, allows to find potential target genes for modulating thousands of traits across a broad set of human tissues. These results are now publicly available and have gathered users across the globe and led to funded collaborations. Most of the associations proved to be tissue-specific, suggesting context specificity of the trait etiology. Significant associations in unexpected tissues also underscored the need for an agnostic scanning of multiple contexts to improve the ability to detect causal regulatory mechanisms. Monogenic disease genes were enriched among significant associations for related traits, suggesting that smaller alterations of these genes may cause a spectrum of milder phenotypes. More details can be found [in this paper](https://www.nature.com/articles/s41467-018-03621-1)
